README.md

```diff
@@ -2,7 +2,9 @@
 
 ## Welcome to thinking inside the box
 Thinking outside the box can be exhausting.
-Maybe it's time to go back to simple.
+
+It's time to go back to simple.
+
 Free your self from distractions with our minimalists thought journal and note-taking application.
```

Comment:
Free yourself from distractions with ThoughtBox, the minimalist thought journal and note-taking system.